CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Mustache library changelog
 
+## v2.3.1
```

Comment:
Version 2.3.1 is already on Hackage. This should be v2.3.2.

Reply:
Oops, fixed now!